AGXUnity/Rendering/DeformableTerrainParticleRenderer.cs

```diff
@@ -33,6 +41,20 @@ public GranuleRenderMode RenderMode
       }
     }
 
+    [Tooltip("Render particles using cloned GameObjects or with Graphics.DrawMeshInstanced")]
```

Comment:
Wrong tooltip and move it to the property `SyncMode` for it to be rendered in the Inspector. Also `RenderMode` above should have the tooltip.